### PR TITLE
Fabo/game of stakes

### PR DESCRIPTION
**Pull request by faboweb**
Prepare for game of stakes release

Needs https://github.com/cosmos/voyager/pull/1794

### CODE REVIEW

**Comment by faboweb** [comment]
No, Voyager was on `0.28.0-rc2`. Which breaking changes were there till now?

**Comment by fedekunze** [comment]
@faboweb redelegation throws an out of gas in location: WriteFlat error

File: app/src/renderer/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -347,74 +347,58 @@ export default {
...
-      let stakingTransactions = {}
-      let txTitle,
-        txBody,
-        txAction = ``
-
-      if (from === delegatorAddr) {
-        txTitle = `delegation`
-        txBody = `delegated`
-        txAction = `delegating`
+      if (from === this.wallet.address) {
```
Comment (sabau) [inline app/src/renderer/components/staking/PageValidator.vue]: Seems to be an improvement on so many level here  👍

File: app/src/renderer/vuex/modules/send.js [hunk 2]
```diff
@@ -37,13 +38,13 @@ export default ({ node }) => {
...
     let to = args.to
     delete args.to
-    args.gas = `50000000`
 
     // submit to LCD to build, sign, and broadcast
     let req = to ? node[type](to, args) : node[type](args)
 
     let res = await req.catch(err => {
       let message
+      console.log(err)
```
Comment (sabau) [inline app/src/renderer/vuex/modules/send.js]: Not sure if it's a leftover or is here on pourpose

Comment (faboweb) [reply app/src/renderer/vuex/modules/send.js]: good catch, fixed

File: app/src/renderer/components/governance/PageProposal.vue [hunk 3]
```diff
@@ -106,7 +106,7 @@
         :show-modal-deposit.sync="showModalDeposit"
         :proposal-id="proposalId"
         :proposal-title="proposal.title"
-        :denom="bondingDenom"
+        :denom="bondDenom"
```
Comment (fedekunze) [inline app/src/renderer/components/governance/PageProposal.vue]:
```suggestion
        :denom="depositDenom"
```

File: app/src/renderer/components/governance/PageProposal.vue [hunk 4]
```diff
@@ -158,13 +158,7 @@ export default {
...
   computed: {
     // TODO: get denom from governance params
-    ...mapGetters([
-      `bondingDenom`,
-      `proposals`,
-      `connected`,
-      `wallet`,
-      `votes`
-    ]),
+    ...mapGetters([`bondDenom`, `proposals`, `connected`, `wallet`, `votes`]),
```
Comment (fedekunze) [inline app/src/renderer/components/governance/PageProposal.vue]:
```suggestion
    ...mapGetters([`depositDenom`, `proposals`, `connected`, `wallet`, `votes`]),
```

File: app/src/renderer/components/governance/PageProposal.vue [hunk 5]
```diff
@@ -261,7 +255,7 @@ export default {
         this.$store.commit(`notify`, {
           title: `Successful deposit!`,
           body: `You have successfully deposited your ${
-            this.bondingDenom
+            this.bondDenom
```
Comment (fedekunze) [inline app/src/renderer/components/governance/PageProposal.vue]:
```suggestion
            this.depositDenom
```

**Comment by fedekunze** [comment]
unit test is failing bc of prettier

**Comment by fedekunze** [comment]
@faboweb and can we update the `README` and remove the no active testnets message ?

**Comment by faboweb** [comment]
@fedekunze delegation doesn't work

**Comment by fedekunze** [comment]
why does it work with bondDenom but not with depositDenom ?

**Comment by fedekunze** [comment]
the submit proposal tx is not displayed and the link to the proposal on the deposit tx is wrong: it says `governance/proposals/1` and it should say `governance/1`

**Comment by fedekunze** [comment]
also unbonding is not working

**Comment by faboweb** [comment]
Merged because coverage was only missing on lcdClientMock which we deprecate.

**Comment by faboweb** [comment]
@fedekunze bug was files #1803

**Pull request merged**